lib/modules/manager/bundler/artifacts.ts

```diff
@@ -93,7 +93,10 @@ export async function updateArtifacts(
 
     const commands: string[] = [];
 
-    if (config.isLockFileMaintenance) {
+    if (
+      config.updateType === 'lockFileMaintenance' ||
```

Comment:
as told, this doesn't work for groups because this field only reflects first update in group which isn't always the lockfile maintenance 


```suggestion
      updatedDeps.some(dep => dep.updateType === 'lockFileMaintenance') ||
```
you need to check the updated deps